opus/application/static_media/css/opus.css

```diff
@@ -170,6 +170,41 @@ ul.searchMenu {
     padding-right: 2em;
 }
 
+/* Add or remove background and checkmark when a search item is active.
+   This is both for the main search tab and the select metadata dialog. */
+.op-search-param-off {
```

Comment:
you should be able to use the bootstrap 4 color classes to add/remove color, no?  I don't think there should be two classes w/all the same except one characteristic.